test/e2e/bigint.2.ts

```diff
@@ -0,0 +1,21 @@
+import { DEFAULT_OPTIONS } from '../../src'
+
+export const input = {
+  title: 'Example Schema -- BigInt Enabled',
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    secondsSinceDawnOfTime: {
+      type: 'integer'
+    },
+    reasonablySizedNumber: {
+      type: 'integer',
+      maximum: 42
```

Comment:
What happens when there's no `minimum`? Can you flesh out the behavior for negative bigints a bit more?